docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -130,9 +132,10 @@ Describe "The 7z bin" {
 Running the tests:
 
 ```powershell
-hab pkg build 7zip
+hab studio run -D "hab pkg build 7zip"
```

Comment:
This might be worthy of a comment line to explain why we invoke `build` like this. Or change this back to be a simple `hab pkg build 7zip`.

Reply:
I'll change this back to original.

Reply:
Amended @james-stocks.